mutable_rules/307_Temporal-Initiated-Motion-Every-Rule.md

```diff
@@ -4,6 +4,6 @@ To ensure the progression of rule proposal, and allow for rule proposal to fail
 
 (1.a) When a turn begins, 48 hours is allotted for the player whose turn it is to propose a rule or amendment. If no proposal is made within this time constraint the turn is forfeit and the proposing player loses 10 points. 
 
-(1.b) 12 hours for discussion and amendment of the proposed rule or amendment. This period is mandatory whether or not discussion occurs.  
+(1.b) 24 hours for discussion and amendment of the proposed rule or amendment before votes are cast. This period is mandatory whether or not discussion occurs.  
 
-(1.d) 24 hours for the players to vote. Votes are tallied either when all players have voted or this time period ends.  During the tally, players who did not vote during this period are considered as having "abstained"
\ No newline at end of file
+(1.e) 24 hours for the players to vote. Votes are tallied either when all players have voted or this time period ends.  During the tally, players who did not vote during this period are considered as having voted to "abstain"
```

Comment:
Oh yeah there's no rule in play here, you can do what you want. Just thought it might have been a mistake. lol